### DISCUSSION TITLE
[duplicate] "Configure Artifact Repository" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12451] [General]
# en/latest/configure-artifact-repository/

https://argo-workflows.readthedocs.io/en/latest/configure-artifact-repository/

**Comment by shabeebk** [comment]
Is the minio instructions still valid? I am getting error while trying to add helm chart for minio following the above instrcutions.
```
$helm repo add minio https://helm.min.io/
Error: looks like "https://helm.min.io/" is not a valid chart repository or cannot be reached: Get "https://helm.min.io/index.yaml": dial tcp: lookup helm.min.io: no such host
```

**Comment by Joibel** [comment]
You need to use https://charts.min.io/ instead, this is in the process of being updated in the docs here https://github.com/argoproj/argo-workflows/pull/12438

**Reply by shabeebk** [reply]
Thanks

**Comment by shabeebk** [comment]
Looks like the link https://argo-workflows.readthedocs.io/en/latest/#:~:text=Access%20the%20demo%20environment is broken. I have got 503 Service Temporarily Unavailable message while trying to access.

**Reply by agilgur5** [reply]
Yea it's been down for some time due to the switch from Intuit infra to CNCF infra. There's an open issue for it: #12157

**Comment by agilgur5** [comment]
Duplicate Giscus discussion of #10320
